lib/modules/cops/cop_template.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Cops
+  class CopTemplate
+    def initialize(file_path, index, warning)
+      @file_path = file_path
+      @index = index
+      @warning = warning
+    end
+
+    def call
+      warning_message(@file_path, @index, @warning)
+    end
+
+    private
+
+    def warning_message(file_path, index, warning)
+      "#{file_path}:#{index + 1}: #{warning}:"
+    end
```

Comment:
I'd add `attr_reader` for  emulator_rubocop, index, warning
and use `warning_message` this way:
```suggestion
    def warning_message
      "#{file_path}:#{index + 1}: #{warning}:"
    end
```

Reply:
Fixed